test/dlc_tests/container_tests/bin/efa/testEFA

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
whats the source for this script?

Reply:
This is taken from DLAMI integration tests for EFA.